libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -495,6 +495,9 @@ public void track(AnalyticsTracker.Stat stat, Map<String, ?> properties) {
             case SITE_SETTINGS_DELETE_SITE_RESPONSE_ERROR:
                 eventName = "site_settings_delete_site_response_error";
                 break;
+            case ABTEST_START:
+                eventName = "abtest_start";
```

Comment:
That's fair. I decided on `start` for consistency between this and Calypso/PHP. I'll defer to your guys' judgement and change it to `started` cc @aerych